### ISSUE TITLE
Imported Volume 

### ISSUE THREAD

**Issue by mscbpi** [PureStorage-OpenConnect/pso-explorer#6]
Imported volume is shown as 0 byte in PSO-Explorer.

This volume was a NFS share exposed to a server that has been converted into a container. The K8s volume was created against this existing NFS share. It had not been created as a K8s volume the first time.

**Comment by sdodsley** [comment]
@mscbpi  Can you provide the output from `kubectl describe pv` and `kubectl describe pvc` so we can check this out?

**Comment by sdodsley** [comment]
Actually, we have reproduced this and know the issue. We have raised an internal RFE to ideal with this.

**Comment by mscbpi** [comment]
In case it helps:

```
Name:            pv-dnabackup
Labels:          <none>
Annotations:     kubectl.kubernetes.io/last-applied-configuration:
                   {"apiVersion":"v1","kind":"PersistentVolume","metadata":{"annotations":{"pv.kubernetes.io/provisioned-by":"pure-csi"},"name":"pv-dnabackup...
                 pv.kubernetes.io/bound-by-controller: yes
                 pv.kubernetes.io/provisioned-by: pure-csi
Finalizers:      [kubernetes.io/pv-protection]
StorageClass:    pure-file
Status:          Bound
Claim:           dna-backup/pvc-dnabackup
Reclaim Policy:  Retain
Access Modes:    RWX
VolumeMode:      Filesystem
Capacity:        100Gi
Node Affinity:   <none>
Message:
Source:
    Type:              CSI (a Container Storage Interface (CSI) volume source)
    Driver:            pure-csi
    VolumeHandle:      DNA_Backup
    ReadOnly:          false
    VolumeAttributes:      backend=file
Events:                <none>

```
```
Name:          pvc-dnabackup
Namespace:     dna-backup
StorageClass:  pure-file
Status:        Bound
Volume:        pv-dnabackup
Labels:        <none>
Annotations:   pv.kubernetes.io/bind-completed: yes
Finalizers:    [kubernetes.io/pvc-protection]
Capacity:      100Gi
Access Modes:  RWX
VolumeMode:    Filesystem
Mounted By:    rsync-ssh-677b8c779-ctmvz
Events:        <none>
```

**Comment by sdodsley** [comment]
@mscbpi we have isolated the issue and are attempting to come up with a resolution. We will close this issue when we have a solution.

**Comment by dnix101** [comment]
@mscbpi We just released 0.5.4 which solved this issue.

**Comment by mscbpi** [comment]
👍 it is indeed working.

Thanks
